apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -616,20 +617,24 @@ export const counterWatcher = counter.stream(
   }
 );
 
-export const counterNamespaceWatcher = counter.stream(
+export const counterNamespaceWatcher = counter.batchStream(
   "counterNamespaceWatch",
   { queryKeys: [{ namespace: "different" }], operations: ["insert"] },
-  async (item) => {
-    console.log(item);
-    const value = await counter.get(item.key);
-    await counter.set({
-      namespace: "default",
-      id: value!.id,
-      n: (value?.n ?? 0) + 1,
-      optional: undefined,
-    });
-    console.log("send signal to", value!.id);
-    await entitySignal.sendSignal(value!.id);
+  async (items) => {
+    await Promise.all(
```

Comment:
Do we support returning partial failures?

Reply:
We do, see below. The batch API needs to handle partial failures manually using the item.id. The per item API you just need to return false.